x-pack/README.md

```diff
@@ -100,6 +100,33 @@ We also have SAML API integration tests which set up Elasticsearch and Kibana wi
 node scripts/functional_tests --config test/saml_api_integration/config
 ```
 
+#### Running Reporting functional tests
+
+prerequisites:
+The reporting functional tests use [pdf-image](https://www.npmjs.com/package/pdf-image) to convert PDF's pages to png files for image comparisions between generated reports and baseline reports.
+pdf-image requires the system commands `convert`, `gs`, and `pdfinfo` to function. Those can be set up be running the following.
+
+```sh
+//OSX
+brew install imagemagick ghostscript poppler
+```
```

Comment:
Might want to add the Linux (Ubutnu) commands too:

`sudo apt-get install imagemagick ghostscript poppler-utils`

I don't see any instructions for Windows, so don't even know if x-pack tests will run there.